wv2util/MainWindow.xaml

```diff
@@ -349,6 +350,42 @@
                 </Grid>
             </TabItem>
 
+            <!-- App Experimental Features List -->
+            <TabItem DataContext="{Binding Source={StaticResource ExperimentalFeatureList}}">
+                <TabItem.Header>
+                    <StackPanel Orientation="Horizontal">
+                        <TextBlock Text="Experimental "/>
+                    </StackPanel>
+                </TabItem.Header>
+                <Grid HorizontalAlignment="Stretch" VerticalAlignment="Stretch">
+                    <ListView x:Name="ExperimentalFeatureList" ItemsSource="{Binding}" Grid.Row="0" Grid.ColumnSpan="2">
+                        <ListView.View>
+                            <GridView>
+                                <GridViewColumn DisplayMemberBinding="{Binding Name}">
+                                    <GridViewColumnHeader>Name</GridViewColumnHeader>
+                                </GridViewColumn>
+                                <GridViewColumn>
+                                    <GridViewColumnHeader>Enabled</GridViewColumnHeader>
+                                    <GridViewColumn.CellTemplate>
+                                        <DataTemplate>
+                                            <Grid Width="{Binding ElementName=CheckBoxColumn, Path=Width}">
+                                                <CheckBox HorizontalAlignment="Center" IsChecked="{Binding IsEnabled, Mode=TwoWay}" />
+                                            </Grid>
+                                        </DataTemplate>
+                                    </GridViewColumn.CellTemplate>
+                                </GridViewColumn>
+                                <GridViewColumn DisplayMemberBinding="{Binding Description}">
+                                    <GridViewColumnHeader>Description</GridViewColumnHeader>
+                                </GridViewColumn>
+                            </GridView>
+                        </ListView.View>
+                    </ListView>
+                </Grid>
+            </TabItem>
+
+            <!--ItemTemplate="{StaticResource AppOverrideListTemplate}"
```

Comment:
What's this?

Reply:
This is the experimental tab for the feature.